### PR TITLE
adds info about servers table and iso generation form

### PR DESCRIPTION
**Pull request by mitchell852**

### CODE REVIEW

File: blueprints/multi-interface-servers.md [hunk 1]
```diff
@@ -529,15 +529,19 @@ If desired, the markup used for that sample may be found in
 
 Additionally, the properties of servers no longer present on a server object
 will need to be removed from table columns (and `interfaces` should not be
-added).
+added). However, if possible, it may make sense to include the ip addresses of
+the one interface designated as the service address in the servers tables.
+
+Also, Traffic Portal's iso generation form will need modification when using the
+'Copy Server Attributes From" dropdown to copy the properties of the one interface
+designated as the service address.
```
Comment (ocket8888) [inline blueprints/multi-interface-servers.md:537]: Is it not possible to configure an ISO with the interface set-up of a multi-interface server?

Comment (mitchell852) [reply blueprints/multi-interface-servers.md:537]: i don't know a lot about isos. i just know the iso generation form gets populated from the ipaddress of the server object (along with other first class server properties). If not modified, that form will not populate after those properties are moved out of the server object.

Comment (mitchell852) [reply blueprints/multi-interface-servers.md:537]: which means the 'Copy Server Attributes From' dropdown will populate less form fields. maybe it's time to remove iso generation entirely from TP?

Comment (ocket8888) [reply blueprints/multi-interface-servers.md:537]: I'm definitely in favor of that, but also what I meant was could the form be updated to not only pull properties from the new field for _one_ interface, but for _all_ interfaces.

Comment (mitchell852) [reply blueprints/multi-interface-servers.md:537]: could it? sure. should it? not sure.

**Pull request merged**